release.sh

```
@@ -37,13 +37,14 @@ version=${tag#v}
 
 git checkout -b $tag
 
-npm version "$version" --no-git-tag-version --allow-same-version
-npm i
+# Bump version in package.json
+bun pm version "$version" --no-git-tag-version
+bun install
 
 # Generate changelog after we updated the version
-npx git-cliff -o CHANGELOG.md -t $tag
+bunx git-cliff -o CHANGELOG.md -t $tag
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)